web/src/message_view.js

```diff
@@ -306,6 +318,7 @@ export function show(raw_terms, opts) {
         raw_terms = [{operator: "is", operand: "home"}];
     }
     const filter = new Filter(raw_terms);
+    filter.try_adjusting_for_moved_with_target();
 
     if (try_rendering_locally_for_same_narrow(filter, opts)) {
```

Comment:
Should `try_rendering_locally_for_same_narrow` do something special with `with` links like it does with `near` links?

Reply:
with the current set of changes, I don't think this would have any relevance with `with` operator